WordPress/Classes/ViewRelated/Reader/ReaderInterestsCoordinator.swift

```diff
@@ -0,0 +1,93 @@
+import Foundation
+
+class ReaderSelectInterestsCoordinator {
+    private struct Constants {
+        static let userDefaultsKeyFormat = "Reader.SelectInterests.hasSeenBefore.%@"
+    }
+
+    private let interestsService: ReaderFollowedInterestsService
+    private let store: KeyValueDatabase
+    private let userId: NSNumber?
+
+    /// Creates a new instance of the coordinator
+    /// - Parameter service: An Optional `ReaderFollowedInterestsService` to use. If this is `nil` one will be created on the main context
+    ///   - store: An optional backing store to keep track of if the user has seen the select interests view or not
+    ///   - userId: The logged in user account, this makes sure the tracking is a per-user basis
+    init(service: ReaderFollowedInterestsService? = nil,
+         store: KeyValueDatabase = UserDefaults.standard,
+         userId: NSNumber? = nil) {
+
+        let defaultContext = ContextManager.sharedInstance().mainContext
+
+        self.interestsService = service ?? ReaderTopicService(managedObjectContext: defaultContext)
+        self.store = store
+        self.userId = userId ?? {
+            let acctServ = AccountService(managedObjectContext: defaultContext)
+            let account = acctServ.defaultWordPressComAccount()
+
+            return account?.userID
+        }()
+    }
+
+    // MARK: - Display Logic
+
+    /// Deteremines whether or not the select interests view should be displayed
```

Comment:
```suggestion
    /// Determines whether or not the select interests view should be displayed
```

Reply:
Fixed in: f222e163204f908ae76cc2aa30495ee02bc98bcd